tribits/core/package_arch/TribitsAdjustPackageEnables.cmake

```diff
@@ -999,14 +999,59 @@ macro(tribits_apply_test_example_enables PACKAGE_NAME)
 endmacro()
 
 
+# Macro to set ${TRIBITS_SUBPACKAGE)_ENABLE_TESTS and
+# ${TRIBITS_SUBPACKAGE)_ENABLE_EXAMPLES based on
+# ${TRIBITS_PARENTPACKAGE)_ENABLE_TESTS or
+# ${TRIBITS_PARENTPACKAGE)_ENABLE_EXAMPLES
+macro(tribits_apply_subpackage_tests_examples_enables  PARENT_PACKAGE_NAME)
+  if ("${${PARENT_PACKAGE_NAME}_ENABLE_EXAMPLES}" STREQUAL ""
+    AND ${PARENT_PACKAGE_NAME}_ENABLE_TESTS
+    )
+    message("-- " "Setting"
+      " ${PARENT_PACKAGE_NAME}_ENABLE_EXAMPLES=${${PARENT_PACKAGE_NAME}_ENABLE_TESTS}"
+      " because"
+      " ${PARENT_PACKAGE_NAME}_ENABLE_TESTS=${${PARENT_PACKAGE_NAME}_ENABLE_TESTS}")
+    set(${PARENT_PACKAGE_NAME}_ENABLE_EXAMPLES ${${PARENT_PACKAGE_NAME}_ENABLE_TESTS})
+  endif()
+  foreach(spkg IN LISTS ${PARENT_PACKAGE_NAME}_SUBPACKAGES)
+    set(fullSpkgName ${PARENT_PACKAGE_NAME}${spkg})
+    if (${PROJECT_NAME}_ENABLE_${fullSpkgName})
+      if (${PARENT_PACKAGE_NAME}_ENABLE_TESTS)
+        if ("${${fullSpkgName}_ENABLE_TESTS}" STREQUAL "")
+          message("-- " "Setting"
+	    " ${fullSpkgName}_ENABLE_TESTS=${${PARENT_PACKAGE_NAME}_ENABLE_TESTS}"
```

Comment:
Not sure how picky you are about whitespace, but you've got some stray tabs here.

Reply:
Initial tabs are a no-no.  Not sure how that got inserted (my emacs editor should be setup to indent with spaces only).   I will diffidently fix that.

Thanks!

Reply:
This is addressed in new PR #515.  I found a bunch of other places where tabs somehow got inserted.  I wish there was a simple linter that could catch this.  (Actually, to look for initial tabs in `*.cmake` files, then that is easy.  I can do that with a simple bash scripts.)